db/fixtures/ae_datastore/ManageIQ/__domain__.yaml

```diff
@@ -6,6 +6,7 @@ object:
     name: ManageIQ
     description: Base domain
     display_name: 
-    system: true
+    system: false
     priority: 0
     enabled: true
+    tenant_id: 1
```

Comment:
@bzwei 
This file should always be excluded from the PR since it changes the locked behavior of ManageIQ domain